src/modules/Dashboard/components/HeaderOptions/HeaderOptions.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { Box, Button } from '@mui/material';
+import { Link, generatePath, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { page } from 'resources';
+import { Svg } from 'shared/components';
+import { ExportDataSetting } from 'shared/features/AppletSettings';
+import { theme, variables } from 'shared/styles';
+import { Mixpanel } from 'shared/utils';
+
+export const HeaderOptions = () => {
+  const [isExportOpen, setIsExportOpen] = useState(false);
+  const { t } = useTranslation('app');
+  const { appletId } = useParams();
+  const isSettingsSelected = location.pathname.includes('settings');
+
+  const handleOpenExport = () => {
+    setIsExportOpen(true);
+    Mixpanel.track('Export Data click');
+  };
+
+  const handleCloseExport = () => {
+    setIsExportOpen(false);
+  };
+
+  return (
+    <Box sx={{ display: 'flex', gap: theme.spacing(1), placeItems: 'baseline' }}>
+      <Button
+        data-testid="header-option-export-button"
+        onClick={handleOpenExport}
+        startIcon={<Svg id="export" width={24} height={24} />}
+        sx={{ color: variables.palette.on_surface_variant }}
+      >
+        {t('export')}
+      </Button>
+
+      <Link
+        data-testid="header-option-settings-link"
+        to={generatePath(page.appletSettings, { appletId })}
+      >
+        <Svg id="settings" fill={isSettingsSelected ? variables.palette.primary : ''} />
+      </Link>
```

Comment:
Just noticed a few little visual discrepancies here:
- I think `StyledFlexTopCenter` may be a better choice here than baseline-aligned items, even though the original code was aligned that way. Also, with `sx`, the theme spacing multiplier is automatically applied to numeric values, so you can just use `sx: {{ gap: 1 }}`
- The `startIcon` SVG should be 18x18 rather than 24x24
- The `Link`'s default `display` of `inline` is causing the SVG to be a little misaligned; I think you can get away with adding `<Link style={{ display: 'flex' }} …` there to fix that (it doesn't support `sx`)
- Bonus: It looks like the settings button is supposed to have a hover style, as it's technically an `icon-button` in Figma. We may not have this component, and not sure if our existing `Button` has icon-only support (with that particular hover style), but just wanted to clock that. May warrant a ticket in the backlog/for followup.

Reply:
I pushed up another commit that should fix the UI issues you described.

Regarding the Figma `icon-button`: I opted to swap to using the `IconButton` from MUI for handling that link. It provides hover/active styles that admittedly aren't themed correctly currently, but it looks like we already have [a task for next sprint](https://mindlogger.atlassian.net/browse/M2-6071) dedicated to updating button variants to match the ones defined in Figma specifically. 

I figure, as you've suggested, adding appropriate theme values for the IconButton component could be a follow-up, and handled as part of that task.

Reply:
Perfect, works for me!